bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/handler/HomematicBridgeHandler.java

```diff
@@ -132,6 +133,10 @@ private void initializeInternal() {
                         ex.getMessage(), ex);
                 disposeInternal();
                 scheduleReinitialize();
+            } catch (ConfigurationException ex) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, ex.getMessage());
+                logger.debug("Configuration error while initialize binding: {}", ex.getMessage(), ex);
```

Comment:
```suggestion
```

As the thing state is allready set, what is the point of logging and writing the stacktrace, is that needed?